sdks/python/apache_beam/tests/pipeline_verifiers_test.py

```diff
@@ -0,0 +1,54 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+"""Unit tests for the test pipeline verifiers"""
+
+import logging
+import unittest
+
+from apache_beam.internal.clients import dataflow
+from apache_beam.runners.runner import PipelineState
+from apache_beam.runners.runner import PipelineResult
+from apache_beam.tests.pipeline_verifiers import PipelineStateMatcher
+from hamcrest import assert_that
```

Comment:
this `assert_that` in has the same name as beam `assert_that`, can you import this with a different name to avoid confusion, maybe `hamcrest_assert_that`.